package proto_ff;

import "yd_fieldoptions.proto";


message E_ConditionConditionConditionDesc
{
	optional int32 m_Param = 1[(yd_fieldoptions.field_cname) = "参数"];
	optional int32 m_Type = 2[(yd_fieldoptions.field_cname) = "类型"];
}

message E_ConditionCondition
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	repeated E_ConditionConditionConditionDesc m_condition = 2[(yd_fieldoptions.field_cname) = "条件", (yd_fieldoptions.field_arysize) = 3];
}


message Sheet_ConditionCondition
{
	repeated E_ConditionCondition E_ConditionCondition_List = 1[(yd_fieldoptions.field_arysize)=32];
}
